README.md

```diff
@@ -108,13 +108,13 @@ public interface BookRepository extends TarantoolRepository<Book, Long> {
 }
 ```
 
-The `@Tuple` annotation allows to specify the space name which schema will be used for forming the data tuples.
-You can add `@Tuple` on the `@Query` annotated method that specifies the space name for a specific stored function call.
-Also, you can add `@Tuple` annotation to the repository - it acts as a shortcut for specifying the tuple return format and space name for all custom methods
-in this repository. The `@Field` annotations provide custom names for the fields. It is necessary to have at least 
+The `@Tuple` annotation allows to specify the space name which schema will be used for forming the data tuples to sending
+and for mapping the incoming tuple into an object.
+You can omit the `@Tuple` annotation, then the connector will take the class name converted to snake_case.
+The `@Field` annotations provide custom names for the fields. It is necessary to have at least 
 one field marked with the `@Id` annotation.
 
-Extending `CrudRepository` causes CRUD methods being pulled into the
+Extending `TarantoolRepository` causes basic CRUD methods being pulled into the
```

Comment:
Why is this changed? CrudRepository is a Spring Data class that the users should inherit from in their applications.

Reply:
In the example above, we inherit from `TarantoolRepository`, which just allows you to use CRUD methods to interact with Tarantool + adds methods from `PagingAndSortingRepository`. Nowhere in the code do we have an example where we inherit from `CrudRepository`, except `README.md`. Therefore, I propose to write `TarantoolRepository` everywhere, so as not to confuse the user, because the interface is the most functional complete, contains the `CrudRepository` and `PagingAndSortingRepository` methods

If you agree, then we should also change here to `TarantoolRepository`
![image](https://user-images.githubusercontent.com/22258443/152527719-be2a0c25-d42b-437b-82b1-028a77e3cc99.png)

Reply:
This breaks the way how using SpringData is intended to look like. The common interfaces like CrudRepository are being substituted by the engine with the most suitable implementation existing in the classpath, TarantoolRepository in this case. Of course, users are allowed to inherit TarantoolRepository directly for using Tarantool-specific methods and features, but in some real cases, the user may have switched from using another implementation (like Redis) or have them both, switching on one or another depending on the environment.

So, I agree that current examples need to be changed for TarantoolRespository, but we need a special example with CrudRepository along with this we the necessary remarks about how it is supposed to work. Also, we need a test that checks CrudRepository example to be working.

Reply:
Thanks, understood the reason for using this interface. Then let's add some examples using `CrudRepository`

Reply:
Returned `CrudRepository` to the README, also added it to the real `BookRepository` code. Added a few words about TarantoolRepository